ansible/environments/local/hosts

```diff
@@ -10,8 +10,11 @@ ansible ansible_connection=local
 controller0         ansible_host=172.17.0.1 ansible_connection=local
 controller1         ansible_host=172.17.0.1 ansible_connection=local
 
-[kafka]
-172.17.0.1          ansible_host=172.17.0.1 ansible_connection=local
+[kafkas]
```

Comment:
we might want to run `local` and travis deployments with two kafka nodes to have better test coverage.

Reply:
Note that travis == local.